zmon-common/src/main/java/org/zalando/zmon/domain/ServiceLevelStatus.java

```diff
@@ -1,5 +1,7 @@
 package org.zalando.zmon.domain;
 
+import java.util.HashMap;
+
 public class ServiceLevelStatus {
```

Comment:
Can you point to the test for this class?

Reply:
we are adding tests case to cover this class now.

Reply:
https://github.com/zalando-zmon/zmon-controller/pull/774/files#diff-c06deda7b963383e3657a0b507718206